docs/sphinx/about_hpx/people.rst

```diff
@@ -230,7 +230,7 @@ the project through discussions, pull requests, documentation patches, etc.
   specific patches for FreeBSD and MSVC12.
 * Agustin Berge, who contributed patches fixing some very nasty hidden template
   meta-programming issues. He rewrote large parts of the API elements ensuring
-  strict conformance with C++11/14.
+  strict conformance with C++11/14/17/20.
```

Comment:
```suggestion
  strict conformance with the C++ ISO standard.
```

Reply:
I just added an upper case to standard as it is appearing on cppreference, any objections?

Reply:
Forgotten push?

Reply:
yes it was on a temporary branch ;)